app/models/host.rb

```diff
@@ -12,4 +14,23 @@ class Host < ApplicationRecord
   has_many :profiles, through: :profile_hosts, source: :profile
 
   validates :name, presence: true, uniqueness: { scope: :account_id }
+
+  class << self
+    def filter_by_compliance(key, operator, value)
```

Comment:
The operator doesn't do anything (am I using this right)?

```rb
> Host.search_for("compliant = false")
[]
> Host.search_for("compliant = true")
[#<Host:0x000055fcaf3d5660 id: "a5a31c32-0bde-4c6f-b69d-f04aaf6529b4", name: "rhel7.localdomain", created_at: Mon, 08 Apr 2019 13:14:22 UTC +00:00, updated_at: Mon, 08 Apr 2019 13:14:22 UTC +00:00, account_id: "2b83f549-8ae8-4352-84e4-b853e2428cef">,
 #<Host:0x000055fcaf3d5390 id: "1e92e15c-0227-4770-acb3-e35a54df6478", name: "rhel7-insights-client.virbr0.akofink-laptop", created_at: Mon, 08 Apr 2019 17:42:32 UTC +00:00, updated_at: Mon, 08 Apr 2019 17:42:32 UTC +00:00, account_id: "2b83f549-8ae8-4352-84e4-b853e2428cef">]
> Host.search_for("compliant != true")
[#<Host:0x000055fcaf3d5660 id: "a5a31c32-0bde-4c6f-b69d-f04aaf6529b4", name: "rhel7.localdomain", created_at: Mon, 08 Apr 2019 13:14:22 UTC +00:00, updated_at: Mon, 08 Apr 2019 13:14:22 UTC +00:00, account_id: "2b83f549-8ae8-4352-84e4-b853e2428cef">,
 #<Host:0x000055fcaf3d5390 id: "1e92e15c-0227-4770-acb3-e35a54df6478", name: "rhel7-insights-client.virbr0.akofink-laptop", created_at: Mon, 08 Apr 2019 17:42:32 UTC +00:00, updated_at: Mon, 08 Apr 2019 17:42:32 UTC +00:00, account_id: "2b83f549-8ae8-4352-84e4-b853e2428cef">]
```

Reply:
You are right, I'll change these two so the operator does something...